clients/drcachesim/tests/offline-kernel-simple.templatex

```diff
@@ -3,14 +3,12 @@ Basic counts tool results:
 Total counts:
  .* total \(fetched\) instructions
  .* total unique \(fetched\) instructions
- .* total non-fetched instructions
```

Comment:
Why did this change?

Reply:
#6752 changed the order. This test's expected output pattern wasn't fixed then because it is not run regularly in our CI.